src/sass/app_2/_common/layout/layouts.scss

```diff
@@ -4,29 +4,76 @@
         height: calc(100vh - #{$HEADER_HEIGHT} - #{$FOOTER_HEIGHT});
         overflow-y: auto;
         overflow-x: hidden;
+        transition: transform 0.25s ease;
+        will-change: transform;
 
         &--show-portfolio-drawer {
-            margin-right: $PORTFOLIO_DRAWER_WIDTH;
+            transform: translateX(-#{$PORTFOLIO_DRAWER_WIDTH});
 
-            @media screen and (max-width: 1280px) {
-                .trade-container {
-                    .chart-container {
-                        grid-column: col-start 1 / span 9;
+            .chart-container {
+                .ciq-chart {
+                    .cq-top-ui-widgets, .cq-chart-controls {
+                        transform: translateX(#{$PORTFOLIO_DRAWER_WIDTH});
+                    }
+                    .cq-top-ui-widgets .info-box {
+                        transform: translateX(-#{$PORTFOLIO_DRAWER_WIDTH});
+                    }
+                }
+                .cq-chart-controls {
+                    will-change: max-width;
+                    max-width: calc(100% - #{$PORTFOLIO_DRAWER_WIDTH});
+                    transform: translateX(#{$PORTFOLIO_DRAWER_WIDTH});
+                }
+                .cq-chart-control-left {
+                    .cq-chart-controls {
+                        transform: translateX(#{$PORTFOLIO_DRAWER_WIDTH}) !important;
+                    }
+                    .info-box {
+                        transform: translateX(-#{$PORTFOLIO_DRAWER_WIDTH}) !important;
                     }
-                    .sidebar-container, .contract-wrapper {
-                        grid-column: col-start 10 / span 3;
+                }
+            }
+            .portfolio, .statement {
```

Comment:
Do we have to update this (and another one on [L#74](https://github.com/binary-com/binary-static/pull/4248/files#diff-d482fe7751bf830c863c2f96d09eacd9R74)) every time adding a new module in the future?

Reply:
Yes, I assume for cashier and others we'll add this as well, maybe we can unify them with a more common class, so that instead of using `margin`, we use `translate` and `max-width` to make the animation consistent with the `trade` app layout.